wallet/src/Pos/Wallet/Web/State/Migrations.hs

```diff
@@ -0,0 +1,16 @@
+-- | This module contains types used for acid-state migrations
+-- that would conflict if declared in the same module.
```

Comment:
Yes, and I confess I didn't do that because I feared that this would have confused `safecopy` (this stem mostly from my ignorance when it comes to this library). I feared that changing a data constructors would have confused the library as that meant messing up with its internal representation. I'm ofc very happy to rename this `WalletTip_V0` I don't see any harm in doing that (on top of my head!)